workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/WorkflowNode.java

```diff
@@ -20,6 +20,7 @@
 public class WorkflowNode {
   private String id;
   private Event event;
+  private String eventId = "tmp";
```

Comment:
maybe the default value here is not necessary, in any case, during the graph construction, the value is going to be replaced by the real event id or the same value as id in here.

Reply:
Yes I should not have committed this one, it was only for my testing